nodes/arduino_handler.py

```diff
@@ -202,15 +203,15 @@ def light_intensity_red_callback(msg): # float 0~1
     actuator_state["light_intensity_red"] = command
 
 
-# The water level sensor is HIGH when dry, which gets passed through the
-# linear_controller node, which takes the sensor value */measured
-# and passes it as */commanded. We should set the pump_5_water_1 to HIGH when
-# we receive a True here. I want to deprecate this with something more
-# feedback loop oriented: https://github.com/OpenAgInitiative/openag_brain/issues/270
-def water_level_high_callback(msg): # Bool
-    command = msg.data
-    if command:
+#PID controller sends us float values ranging from very small to almost 1.0.
```

Comment:
For future reference the water level is not connected to a PID controller but it's connected to a direct_controller in the current launchfile configuration.